test/images/sample-apiserver/VERSION

```diff
@@ -1 +1 @@
-1.10
+1.11
```

Comment:
as the code is based on `1.10` release, lets bump this version some something meaningful like `1.10v1`

Reply:
But when we build these now, they will be built against 1.15... In general the versioning system doesn't make sense in this directory.

Reply:
na! this image is little different, I see `1.10` code is checked out in the dockerfile here - https://github.com/kubernetes/kubernetes/blob/master/test/images/sample-apiserver/Dockerfile#L26